mantidproject/mantid
# Applied clang-tidy modernize-use-using fixes for Framework.

Maintainance PR forming part of #22048.
Applied modernize-use-using fixes for `Framework`.

**To test:**
1. Ensure builds pass.
2. Check for occurrences of typedef not in comments using `grep -rnI '^[^/*]*typedef'` or a similar command of your choice.

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
